src/components/themed/ModalParts.tsx

```diff
@@ -61,6 +68,8 @@ export function ModalFooter(props: ModalFooterProps) {
   )
 }
 
+ModalFooter.bottomRem = 2.5
```

Comment:
Oh interesting. What's the benefit for exporting this value as a static property? Usually we export some relevant constants from the module.

Reply:
It's a little funky, but cute.